thunx-bom/build.gradle

```diff
@@ -0,0 +1,21 @@
+plugins {
+    id 'java-platform'
+    id 'maven-publish'
+}
+
+dependencies {
+    constraints {
+        api project(':thunx-model')
```

Comment:
My question here is: is everything an API dependency?

Aren't some parts only runtime dependencies? 

https://docs.gradle.org/current/userguide/java_platform_plugin.html#sec:java_platform_separation

Reply:
I was wondering about that too.

1. If you use auto-configuration, everything is just a runtime dependency.
2. When you want to override the auto-configuration, nothing is ...

Reply:
Then they all are API dependencies indeed.